groups/alteryx-sandbox/ec2_server.tf

```diff
@@ -25,6 +25,13 @@ module "alteryx_server_ec2_security_group" {
       protocol    = "tcp"
       description = "WMI Access"
       cidr_blocks = join(",", local.azure_dc_cidrs)
+    },
+    {
+      from_port   = 5986
+      to_port     = 5986
+      protocol    = "tcp"
+      description = "Ansible Access"
+      cidr_blocks = join(",", local.internal_cidrs)
```

Comment:
It would be beneficial to have this defined as a new security group and set of security group rule resources for future flexibility.

Also we need to consider what the source of the WinRM connections will be - i.e. is local admin access all that is required, or do we need access from Concourse as well?

Finally, it would be worth considering using the admin prefix list, rather than specifying the CIDR list.